src/main/java/de/tum/in/www1/artemis/service/LearningObjectService.java

```diff
@@ -0,0 +1,40 @@
+package de.tum.in.www1.artemis.service;
+
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.LearningObject;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.lecture.LectureUnit;
+import de.tum.in.www1.artemis.domain.lecture.LectureUnitCompletion;
+
+@Service
+public class LearningObjectService {
```

Comment:
Javadoc would be nice, and maybe a few words what a learning object is because i've never heard that term in the artemis context.